packages/common/src/ui/components/buttons/standard/BaseButton.tsx

```diff
@@ -14,7 +14,7 @@ export const StyledBaseButton = styled(MuiButton)(
       }
 
       if (color === 'primary') {
-        return 'rgb(163, 64, 33)';
+        return theme.palette.primary.main;
```

Comment:
almost offroad.. it is important when using a custom theme. also the current colour is darker than I'd expect, I tried the primary.dark but thought that too dark as well. 🤷